install_config/install/advanced_install.adoc

```diff
@@ -2382,10 +2394,27 @@ xref:running-the-advanced-installation-individual-components[Running Individual
 === Running the RPM-based Installer
 
 The RPM-based installer uses Ansible installed via RPM packages to run playbooks
-and configuration files available on the local host. To run the installer, use
-the following command, specifying `-i` if your inventory file located somewhere
-other than *_/etc/ansible/hosts_*:
+and configuration files available on the local host.
+
+To run the RPM-based installer:
+
+. Run the *_prequisites.yml_* playbook. This must be run only once before
+deploying a new cluster. Use the following command, specifying `-i` if your
+inventory file located somewhere other than *_/etc/ansible/hosts_*:
++
+----
+ifdef::openshift-enterprise[]
+# ansible-playbook  [-i /path/to/inventory] \
+    /usr/share/ansible/openshift-ansible/playbooks/prerequisites.yml
+endif::[]
+ifdef::openshift-origin[]
+# ansible-playbook [-i /path/to/inventory] \
+    ~/openshift-ansible/playbooks/prerequisites.yml
+endif::[]
+----
```

Comment:
Should this not be part of the host preparation? and be a link from this doc?

Reply:
They need to have crafted their inventory file to run this, which is covered in the Advanced Installation topic.